core/translate.py

```diff
@@ -121,14 +122,12 @@ def _get_json5_from_google(self, text):
         return json
 
     def _get_translation_from_json5(self, content):
-        result = ""
-        pos = 2
-        while True:
-            m = self.match_string.match(content.decode('utf-8'), pos)
-            if not m:
-                break
-            result += m.group(1)
-            pos = m.end()
+        #print(content.decode('utf-8'))
+        response = content.decode('utf-8')
+        fixedJSON = re.sub(r',{2,}', ',', response).replace(',]', ']')
+        data = json.loads(fixedJSON)
+        #print(json.dumps(data, sort_keys=False, indent=2, separators=(',', ': ')))
+        result = data[0][0][0]
```

Comment:
If you have multiple sentences highlighted, this only returns the first one.

Reply:
Thanks you..
Replace content from this file "translate.py" for:

[content](https://raw.githubusercontent.com/MTimer/SublimeText-Google-Translate-Plugin/8ec71ac7976328b6c4d2152b629d1a98423c0467/core/translate.py)